Manito/Manito/Screens/Interaction/Cell/OpenManittoCollectionViewCell.swift

```diff
@@ -36,17 +36,13 @@ final class ManittoCollectionViewCell: BaseCollectionViewCell {
     
     // MARK: - func
     
-    func setManittoCell(with manittoTypeIndex: Int) {
-        self.backgroundColor = Character.allCases[manittoTypeIndex].color.withAlphaComponent(0.5)
+    func configureCell(colorIndex: Int) {
+        self.backgroundColor = Character.allCases[colorIndex].color.withAlphaComponent(0.5)
+        self.characterImageView.image = Character.allCases[colorIndex].image
         self.contentView.alpha = 0.5
-        self.characterImageView.image = Character.allCases[manittoTypeIndex].image
     }
     
-    func setHighlightCell(with manittoTypeIndex: Int, matchIndex: Int, imageIndex: Int) {
-        if manittoTypeIndex == matchIndex {
-            self.backgroundColor = Character.allCases[imageIndex].color
-            self.contentView.alpha = 1.0
-            self.characterImageView.image = Character.allCases[imageIndex].image
-        }
+    func highlightCell() {
+        self.contentView.alpha = 1.0
     }
```

Comment:
p1;

현재 highlight될 때 셀의 뒷 배경의 색깔이 흐리게 나옵니다.

원래는 선택되는 셀의 뒷 배경도 진하게 나와야합니다.

https://user-images.githubusercontent.com/78677571/230859851-63870c19-d94a-44ea-bd0e-7a4597de4ac3.mp4


제가 수정한 방법은
```suggestion
    func highlightCell(colorIndex: Int) {
        self.backgroundColor = Character.allCases[colorIndex].color
        self.contentView.alpha = 1.0
    }
```

파라미터를 추가해주고 

```swift
// OpenManittoViewController.swift


func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
    let cell: OpenManittoCollectionViewCell = collectionView.dequeueReusableCell(forIndexPath: indexPath)

    if let colorIndex = self.friendsList.members?[indexPath.item].colorIndex {
        cell.configureCell(colorIndex: colorIndex)
        if indexPath.item == self.openManittoView.randomIndex {
            cell.highlightCell(colorIndex: colorIndex)
        }
    }

    return cell
}
```

여기서에 if let안에 
```swift
        if indexPath.item == self.openManittoView.randomIndex {
            cell.highlightCell(colorIndex: colorIndex)
        }
```
위 코드를 넣어줬습니다.

꼭 위 방법이 아니더라도 다시 뒷 배경의 색이 들어와야 할 것 같네요!

Reply:
[2fcbc67](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/433/commits/2fcbc67fc4f75eed35b9a8ba32dd79283eab4ed1)
문제를 인지하고 수정했습니다! 감사합니다!
위에서 알려주신 방법으로 하려고 했으나, 제가 저 부분을 구현할 때 `alpha`만으로 셀의 하이라이트를 구현하고자 한거라서 그렇게 될 수 있도록 다른 부분들을 수정했습니다. 

contentView에 alpha를 변경한 것이기 때문에 나머지 부분들이 contentView에 설정되도록 수정했습니다.

**[ 변경된 화면 ]**

https://user-images.githubusercontent.com/55099365/232507615-bd15e7a4-ff5d-4390-96a1-6520bd79cf98.mp4

Reply:
좋습니다! ^_______^